<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #E44B4B}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #4B92E4}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #6BD761}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #F6A051}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #DB86E9}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>20</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>20</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>20</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #B9B4B0}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="velocidade_trem1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="velocidade_trem2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="velocidade_trem3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="velocidade_trem4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="velocidade_trem5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_velocidade_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #E44B4B}</string>
    </property>
    <property name="text">
     <string>  T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_velocidade_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #4B92E4}</string>
    </property>
    <property name="text">
     <string>  T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_velocidade_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #6BD761}</string>
    </property>
    <property name="text">
     <string>  T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_velocidade_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #F6A051}</string>
    </property>
    <property name="text">
     <string>  T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_velocidade_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #DB86E9}</string>
    </property>
    <property name="text">
     <string>  T5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zerar_trem1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Zerar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zerar_trem2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Zerar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zerar_trem3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Zerar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zerar_trem4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Zerar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zerar_trem5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Zerar</string>
    </property>
   </widget>
   <zorder>label_trilho4_7</zorder>
   <zorder>label_trilho1_8</zorder>
   <zorder>label_trilho4_6</zorder>
   <zorder>label_trilho1_7</zorder>
   <zorder>label_trilho4_8</zorder>
   <zorder>label_trilho4_5</zorder>
   <zorder>label_trilho1_5</zorder>
   <zorder>label_trilho4_4</zorder>
   <zorder>label_trilho1_6</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho1_4</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trem1</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_trem2</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trem4</zorder>
   <zorder>label_trem5</zorder>
   <zorder>velocidade_trem1</zorder>
   <zorder>velocidade_trem2</zorder>
   <zorder>velocidade_trem3</zorder>
   <zorder>velocidade_trem4</zorder>
   <zorder>velocidade_trem5</zorder>
   <zorder>label_velocidade_1</zorder>
   <zorder>label_velocidade_2</zorder>
   <zorder>label_velocidade_3</zorder>
   <zorder>label_velocidade_4</zorder>
   <zorder>label_velocidade_5</zorder>
   <zorder>zerar_trem1</zorder>
   <zorder>zerar_trem2</zorder>
   <zorder>zerar_trem3</zorder>
   <zorder>zerar_trem4</zorder>
   <zorder>zerar_trem5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
